### DISCUSSION TITLE
More options to animate group layers….

### DISCUSSION THREAD

**Discussion by amoraleite** [keiwando/toonsquid#526] [Ideas]
Hi!

At the moment we can only animate the opacity on a group layers. Would be great if we could animate position, scale and all other parameters... 

Thanks!

**Comment by RockmanEX3** [comment]
I agree, especially for the drawing layers as keyframing only works on one of the drawing layers rather than all.  Granted, there is a workaround via using “symbols” … though it does take a while to use.  Going back and forth from drawing to keyframing without having to hop from symbol to symbol would be a huge time saver.

**Reply by DWrightAnimation** [reply]
It's said that feature would be coming in the 2.0 update.
